### PR TITLE
Feat - WIP-321 - Validate the answer length in the front-end

### PR DESCRIPTION
**Pull request by rlara19**
#### What does this PR do?
Validate the maximum length of 3000 characters for answers. A warning appears if the submission is longer than allowed showing the maximum length.

#### Where should the reviewer start?
src/components/SubmitAnswerModel.js

#### How should this be manually tested?
Test with shorter and longer answers than the allowed amount of characters.

#### Any background context you want to provide?
https://wizeline.atlassian.net/browse/WIP-319

#### What are the relevant tickets?
https://wizeline.atlassian.net/browse/WIP-321

### CODE REVIEW

File: src/components/SubmitAnswerModal.js [hunk 1]
```diff
@@ -18,6 +18,11 @@ class SubmitAnswerModal extends React.Component {
   submitAnswer = (e) => {
     e.preventDefault();
     const answerValue = this.answerInput.value;
+    const maximumAnswerLength = 3000;
```
Comment (isa-bel) [inline src/components/SubmitAnswerModal.js]: Why 3000?

Comment (rlara19) [inline src/components/SubmitAnswerModal.js]: We agreed that length with @arrutneo and @diegospd. Maybe I should add this link: https://wizeline.atlassian.net/browse/WIP-319

**Comment by alan-maldonado** [comment]
I think you should set a prop to `textarea` component to allow max 3000 characters.

File: src/components/SubmitAnswerModal.js (hunk 1 above)
Comment (alan-maldonado) [inline src/components/SubmitAnswerModal.js]: I think you can set `maxAnswerLength ` to a default prop

Comment (rominavarela) [inline src/components/SubmitAnswerModal.js]: Please put that value in a constants file. Then import it. I think @diegospd was doing that too.

**Pull request merged**